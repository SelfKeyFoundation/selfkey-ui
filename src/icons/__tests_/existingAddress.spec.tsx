import * as React from 'react';
import { create } from 'react-test-renderer';
import ExistingAddressIcon from '../existing-address';

describe('Existing Address Icon', () => {
  it('should show Existing Address Icon', () => {
    const component = create(
      <ExistingAddressIcon />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});