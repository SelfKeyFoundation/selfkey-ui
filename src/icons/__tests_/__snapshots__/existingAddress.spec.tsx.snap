// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Existing Address Icon should show Existing Address Icon 1`] = `
<svg
  aria-hidden="true"
  className="MuiSvgIcon-root"
  focusable="false"
  style={
    Object {
      "height": "25px",
      "width": "28px",
    }
  }
  viewBox="0 0 28 25"
>
  <defs />
  <g
    fill="none"
    fillRule="evenodd"
    id="ðŸ› -Wallet-Setup"
    stroke="none"
    strokeWidth="1"
  >
    <g
      fill="#23E6FE"
      fillRule="nonzero"
      id="111---Import-Keystore-File-(UTC)"
      transform="translate(-346.000000, -399.000000)"
    >
      <g
        id="Modal"
        transform="translate(240.000000, 268.000000)"
      >
        <g
          id="Buttons"
          transform="translate(40.000000, 106.000000)"
        >
          <g
            id="button1"
          >
            <path
              d="M66,25 L66,27.2727273 L93.2727273,27.2727273 L93.2727273,25 L66,25 Z M66,34.0909091 L66,36.3636364 L93.2727273,36.3636364 L93.2727273,34.0909091 L66,34.0909091 Z M87.5909091,38.6363636 L85.7731716,42.7268284 L81.9090909,43.1818182 L84.75,45.9095352 L84.1818182,50 L87.5909091,47.7272727 L91,50 L90.4318182,45.9095352 L93.2727273,43.1818182 L89.2954545,42.7268284 L87.5909091,38.6363636 Z M66,43.1818182 L66,45.4545455 L79.6363636,45.4545455 L79.6363636,43.1818182 L66,43.1818182 Z"
              id="icon-existing"
            />
          </g>
        </g>
      </g>
    </g>
  </g>
</svg>
`;
